layers/+emacs/org/packages.el

```diff
@@ -544,12 +544,33 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (kbd "s-M-SPC") 'spacemacs/org-agenda-transient-state/body)))
 
 (defun org/init-org-brain ()
+  (use-package ascii-art-to-unicode
```

Comment:
Hmmm I don't like the implicit loading of `ascii-art-to-unicode` package in your "init" function. Please add a separate init function for the package and mention it in  `org-packages`. This would otherwise not trigger auto installation for the package.

It should be something of the line:
```suggestion
  (setq org-packages
      '(
        (ascii-art-to-unicode :toggle (version<= "25" emacs-version))
        company
        company-emoji
        emoji-cheat-sheet-plus
        evil-org
        evil-surround
        gnuplot
        (helm-org-rifle :toggle (configuration-layer/layer-used-p 'helm))
        htmlize
        ;; ob, org and org-agenda are installed by `org-plus-contrib'
        (ob :location built-in)
        (org :location built-in)
        (org-agenda :location built-in)
        (org-brain :toggle (version<= "25" emacs-version))
        (org-expiry :location built-in)
        (org-journal :toggle org-enable-org-journal-support)
        org-download
        org-mime
        org-pomodoro
        org-present
        (org-projectile :requires projectile)
        (ox-twbs :toggle org-enable-bootstrap-support)
        ;; use a for of ox-gfm to fix index generation
        (ox-gfm :location (recipe :fetcher github :repo "syl20bnr/ox-gfm")
                :toggle org-enable-github-support)
        (ox-reveal :toggle org-enable-reveal-js-support)
        persp-mode
        (ox-hugo :toggle org-enable-hugo-support)
        (org-trello :toggle org-enable-trello-support)
        ))
(defun org/init-ascii-art-to-unicode ()
  (use-package ascii-art-to-unicode
    :defer t
    :init
    (add-hook 'org-brain-after-visualize-hook #'aa2u-buffer)))

(defun org/init-org-brain ()
  (use-package org-brain
    :defer t
    :init
    (progn
      (spacemacs/declare-prefix "aoB" "org-brain")
      (spacemacs/set-leader-keys
        "aoBv" 'org-brain-visualize
        "aoBa" 'org-brain-agenda)

      (spacemacs/declare-prefix-for-mode 'org-mode "mB" "org-brain")
      (spacemacs/set-leader-keys-for-major-mode 'org-mode

        "Bac" 'org-brain-add-child
        "Bap" 'org-brain-add-parent
        "Baf" 'org-brain-add-fiendship

        "Bgc" 'org-brain-goto-child
        "Bgp" 'org-brain-goto-parent
        "Bgf" 'org-brain-goto-friend

        "BR"  'org-brain-refile
        "Bx"  'org-brain-delete-entry)
      (evil-set-initial-state 'org-brain-visualize-mode 'emacs))))
```